conu/utils/__init__.py

```diff
@@ -123,13 +123,13 @@ def run_cmd(cmd, return_output=False, ignore_status=False, **kwargs):
     """
     logger.debug('command: "%s"' % ' '.join(cmd))
     try:
+        process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
+                                   universal_newlines=True, **kwargs)
+        output = process.communicate()[0]
+        logger.debug(output)
         if return_output:
-            return subprocess.check_output(cmd,
-                                           stderr=subprocess.STDOUT,
-                                           universal_newlines=True,
-                                           **kwargs)
-        else:
-            subprocess.check_call(cmd, **kwargs)
```

Comment:
you should also check for return code of the command; if it's `> 0`, an exception should be raised

Reply:
Okay got it. That makes sense.